RotaryInvertedPendulum-arduino/PIDControl/PIDControl.ino

```diff
@@ -59,12 +59,110 @@ State state = WAITING;
 int counterPlot = 0;
 int frequencyPlot = 20;
 
+bool can_print = false;
+
+void tare_pendulum_encoder()
+{
+  pendulum_initial_position = 0.0;
+  pendulum_initial_position = convertRawAngleToDegrees();
+}
+
+// Compute the exponential smoothing coefficient between 0.0 and 1.0.
+// 0.0: No filter. Only update value is used.
+// 1.0: No update. Only state value is used.
+double alpha_from_freq(double freq, double dt)
```

Comment:
Can you add a description for what `freq` and `dt` should be ? `dt` seems to be the elapsed time in seconds, but I couldn't understand why this is called with `freq` = 500 Hz.

Reply:
Addressed in 509e2a2ed3cfdc8efffe57a21d5cedde4854543e